rfd/0139-gogo-must-go.md

```diff
@@ -276,6 +276,40 @@ extensions from `types.proto`. As they'll no longer have an impact on Teleport's
 logic, the impact should be contained to modifying conversion functions. To handle the new
 output of the generated objects.
 
+#### New objects
+
+New objects can use the internal object approach above, or alternatively can rely on `protojson`'s
+encoding of the protobuf resources. When using `protojson`, we should make sure to use the `UseProtoNames`
+marshaling option to ensure that the marshaled resource is using snake case. If a developer
```

Comment:
`protojson` supports the `json_tag` native option from protobuf

```protobuf
// Trait is a trait that can be use in various resources.
message Trait {
  // key is the name of the trait.
  string key = 1;
  // values is the list of trait values.
  repeated string values = 2;
}

// Trait is a trait that can be use in various resources.
message CustomTrait {
  // key is the name of the trait.
  string key = 1;
  // values is the list of trait values.
  repeated string values = 2 [json_name = "custom_value"];
}


```

```
Trait:
{"key":"key", "values":["value1", "value2"]}

CustomTrait
{"key":"key", "custom_value":["value1", "value2"]}

```

Reply:
I am not saying we should use them, but it might be worth mentioning for cases where we are forced to rename existing fields 

This feature should be supported in every language that allows converting proto -> json using protojson like packages